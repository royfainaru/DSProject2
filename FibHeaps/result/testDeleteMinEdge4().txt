╮
├── ╭────────────────╮
│   │ Key: 3         │
│   ├────────────────┤
│   │ Rank: 1        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 2        │
│   │ Prev: (null)   │
│   │ Child: 4       │
│   ╰────────────────╯
│   ╰── ╭───────────────╮
│       │ Key: 4        │
│       ├───────────────┤
│       │ Rank: 0       │
│       │ Marked: false │
│       │ Parent: 3     │
│       │ Next: (null)  │
│       │ Prev: (null)  │
│       │ Child: (null) │
│       ╰───────────────╯
╰── ╭────────────────╮
    │ Key: 2         │
    ├────────────────┤
    │ Rank: 0        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: (null)   │
    │ Prev: 3        │
    │ Child: (null)  │
    ╰────────────────╯
