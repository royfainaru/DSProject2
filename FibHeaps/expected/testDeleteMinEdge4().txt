╮
├── ╭────────────────╮
│   │ Key: 2         │
│   ├────────────────┤
│   │ Rank: 0        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 3        │
│   │ Prev: 3        │
│   │ Child: (null)  │
│   ╰────────────────╯
╰── ╭────────────────╮
    │ Key: 3         │
    ├────────────────┤
    │ Rank: 1        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: 2        │
    │ Prev: 2        │
    │ Child: 4       │
    ╰────────────────╯
    ╰── ╭───────────────╮
        │ Key: 4        │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 3     │
        │ Next: 4       │
        │ Prev: 4       │
        │ Child: (null) │
        ╰───────────────╯
